aten/src/ATen/cpu/vec256/vec256_double.h

```diff
@@ -238,6 +238,19 @@ Vec256<double> inline operator^(const Vec256<double>& a, const Vec256<double>& b
   return _mm256_xor_pd(a, b);
 }
 
+template <>
+void convert(const double* src, double* dst, int64_t n) {
```

Comment:
THDoubleVector_copy_AVX

Reply:
What is convert doing here? It doesn't like it's actually changing a type? This looks more like an assignment or set. I think set might also be able to do what you're doing here.

Reply:
@cpuhrsch It's a "conversion" from T to T, a.k.a. a copy. It's... not an unreasonable way of spelling it, I suppose, but a bit indirect.